test/test_pep518.py

```diff
@@ -24,7 +28,8 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "requests==2.23.0"
+    "requests==2.22.0; python_version<'3.6'",
```

Comment:
Maybe add some extra spaces to the test? I'm not sure if it matter a lot, but that's what's reported in #392.

Reply:
There's a space (and even worse, a semicolon) in the string above. But I can add spaces around the setuptools too, just to be sure.

Reply:
Ah, yeah, sorry. Missed that!

Reply:
Added spaces above. :)